tests/helpers.test.ts

```diff
@@ -0,0 +1,106 @@
+jest.unmock('../src/helpers')
+jest.mock('fs')
+
+import { pathToJestPackageJSON } from '../src/helpers'
+import { existsSync } from 'fs'
+import * as path from 'path'
+
+const defaultPathToJest = 'node_modules/.bin/jest'
+
+describe('ModuleHelpers', () => {
+  describe('pathToJestPackageJSON', () => {
+    it('should return null when not found', () => {
+      ;(existsSync as jest.Mock<boolean>).mockReturnValueOnce(false).mockReturnValueOnce(false)
```

Comment:
I'd take `;(existsSync as jest.Mock<boolean>)` and move it to line for 

`const existsMock = existsSync as jest.Mock<boolean>` or `const existsMock = existsSync as any as jest.Mock<boolean>` if you have to

Reply:
That sounds good. 👍